python/clean_database.py

```diff
@@ -0,0 +1,33 @@
+from cdp_backend.database import models as db_models
+import fireo
+from google.auth.credentials import AnonymousCredentials
+from google.cloud.firestore import Client
+import requests
+import sys
+
+# Connect to the database
+fireo.connection(client=Client(
+    project="cdp-san-jose-5d9db455",
+    credentials=AnonymousCredentials()
+))
+
+sessions_for_delete = []
```

Comment:
This is just a minor suggestion. I would think we are guaranteed to get unique sessions in `Session.collection.fetch`, but it would be explicit if we use a set here. e.g. `sessions_for_delete = set()` here, then `sessions_for_delete.add(session)` below.

Reply:
On second thought I'm pretty sure we are guaranteed of unique entries because of our id hashing. So please disregard this.